bin/search_solr_tools

```diff
@@ -12,6 +13,47 @@ class SolrHarvestCLI < Thor
     puts SearchSolrTools::VERSION
   end
 
+  desc 'errcode', 'Get a description of the specified error code'
+  def errcode(code)
```

Comment:
Where is this method actually used?

Reply:
I will document this, and add an option for not supplying a code at all which will show all error codes.  I think allowing for a code to be submitted could be helpful so that the user doesn't have to try and do the binary math, so it will only show the codes that apply to a given number.